src/uct/ib/base/ib_device.h

```diff
@@ -55,6 +55,14 @@ enum {
 };
 
 
+enum {
```

Comment:
pls add typedef for this enum and use it instead of int for passing roce version